appveyor.yml

```diff
@@ -12,6 +12,7 @@ environment:
 
 install:
   - ps: Install-Product node $env:nodejs_version
+  - npm install --save fs-extra
```

Comment:
As I understand now, problem was that older version stuck in cache. I proposed to return the cache but invalidate it if package.json have changed.